### PR TITLE
Replaced Page class with Json #8 

### PR DESCRIPTION
**Pull request by diliger**
merged StandalonePage with LauncherWrapperPage to MasterPage

### CODE REVIEW

File: .vs/config/applicationhost.config [hunk 1]
```diff
@@ -0,0 +1,1030 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (warpech) [inline .vs/config/applicationhost.config]: Please remove the `.vs/config/applicationhost.config` file from the repo. It contains private Visual Studio configuration. Consider replacing the `.gitignore` file with the one from StarcounterPrefabs/Guidelines - it has a rule to ignore these files.

File: src/Skyper/Api/MainHandlers.cs [hunk 2]
```diff
@@ -46,8 +49,8 @@ internal class MainHandlers {
             });
 
             Handle.GET("/skyper/contact/{?}", (string objectId) => {
-                return Db.Scope<StandalonePage>(() => {
-                    StandalonePage master = this.GetMaster();
+                return Db.Scope<MasterPage>(() => {
```
Comment (warpech) [inline src/Skyper/Api/MainHandlers.cs]: As @aloscha pointed out in some other code review, it does not really make sense to use `Db.Scope<T>(() => { ... })` in most cases. Could you please replace all occurrences with  `Db.Scope(() => { ... })` ?

Comment (aloscha) [inline src/Skyper/Api/MainHandlers.cs]: Agreed @warpech

File: src/Skyper/Api/MainHandlers.cs [hunk 3]
```diff
@@ -5,36 +5,39 @@
...
+
             Handle.GET("/skyper/standalone", () => {
                 Session session = Session.Current;
 
                 if (session != null && session.Data != null) {
                     return session.Data;
                 }
 
-                StandalonePage standalone = new StandalonePage();
+                MasterPage master = new MasterPage();
```
Comment (aloscha) [inline src/Skyper/Api/MainHandlers.cs]: @warpech so here we should use `var`

**Comment by warpech** [comment]
This comment still needs to be applied: https://github.com/StarcounterSamples/Skyper/pull/9#discussion_r96017812

**Comment by warpech** [comment]
Thanks, merging.

**Pull request merged**